app/src/androidTest/java/net/skyscanner/backpack/BpkResources.kt

```diff
@@ -28,4 +28,15 @@ class BpkResources {
    assertEquals("#ff817b8f", appContext.getResources().getString(R.color.bpkGray500));
    assertEquals("#ffffffff", appContext.getResources().getString(R.color.bpkWhite));
   }
+
+  @Test
+  fun bpkSpacing() {
+    val appContext = InstrumentationRegistry.getTargetContext()
+    assertEquals("4.0dip", appContext.getResources().getString(R.dimen.bpkSpacingSm));
```

Comment:
note `dip` is Density-independent Pixels value that android returns from the android resources